libs/ts-rest/express/src/lib/ts-rest-express.spec.ts

```diff
@@ -1,4 +1,51 @@
-import { getValue } from './ts-rest-express';
+import { initContract } from '@ts-rest/core';
+import { getValue, initServer } from './ts-rest-express';
+
+const c = initContract();
+const postsRouter = c.router({
+  getPost: {
+    method: 'GET',
+    path: `/posts/:id`,
+    responses: {
+      200: null,
+    },
+  },
+});
+
+describe('strict mode', () => {
+  it('allows unknown responses when not in strict mode', () => {
+    const cLoose = c.router({ posts: postsRouter });
+    const s = initServer();
+
+    s.router(cLoose, {
+      posts: {
+        getPost: async ({ params: { id } }) => {
+          return {
+            status: 201,
+            body: null,
+          };
+        },
+      },
+    });
+  });
+
+  it('does not allow unknown statuses when in strict mode', () => {
+    const cStrict = c.router({ posts: postsRouter }, { strict: true });
+    const s = initServer();
+
+    s.router(cStrict, {
+      posts: {
+        // @ts-expect-error 201 is not defined as a known response
```

Comment:
I wanted to test all the types combined so used this approach. Let me know if you would prefer the `Expect<Equal<>>` approach instead.

Reply:
This approach makes sense here, so all good!